compose/cli/main.py

```diff
@@ -545,12 +545,14 @@ def up(self, project, options):
             --no-recreate          If containers already exist, don't recreate them.
                                    Incompatible with --force-recreate.
             --no-build             Don't build an image, even if it's missing
+            --cascade-stop         Stops all containers if any container was stopped.
```

Comment:
- We've got a real verbosity/clarity problem here, yeah. Is "on-container-exit" a good compromise?
- Perhaps "abort" instead of "shutdown" or "stop"?

```
docker-compose up --abort-on-container-exit
```